partials/_body.scss

```diff
@@ -0,0 +1,89 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  display: grid;
+  grid-template-rows: 50px 1fr;
+  grid-template-columns: 1fr;
+  height: 100vh;
+}
+
+input {
+  border: 1px solid black;
+}
+
+button {
+  border: 1px solid black;
+}
+
+main {
+  display: grid;
+  grid-template-rows: 150px 1fr;
+  grid-template-columns: 1fr;
+}
+.result-area {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  input {
+    height: 40%;
+    width: 60%;
+  }
+}
+
+.result-area {
```

Comment:
.result-area is declared two times. Line 28 and 40

Reply:
thank you! just fixed it